src/utils/selectors.ts

```diff
@@ -254,3 +209,7 @@ export const noopSelector: SparkleSelector<undefined> = () => undefined;
 
 export const emptyArray = [];
 export const emptyArraySelector = <T>(): T[] => emptyArray;
+
+export const ownedVenuesSelector: SparkleSelector<
+  Record<string, AnyVenue> | undefined
+> = (state) => state.firestore.data.ownedVenues;
```

Comment:
```suggestion
export const ownedVenuesSelector: SparkleSelector<
  Record<string, AnyVenue> | undefined
> = (state) => state.firestore.ordered.ownedVenues;
```

Also, add the data selector

Reply:
now with 2 selectors 91bb8cbdd6f157e7c45f103389ebc034a5cdfa35